--- app.js-orig	2014-10-16 02:08:12.332299533 -0400
+++ app.js	2014-10-16 02:24:59.415229476 -0400
@@ -15,30 +15,26 @@
 var
   http    = require( 'http'    ),
   express = require( 'express' ),
+  morgan = require( 'morgan' ),
+  bodyParser = require( 'body-parser' ),
+  methodOverride = require( 'method-override' ),
+  errorHandler = require( 'errorhandler' ),
 
   app     = express(),
   server  = http.createServer( app );
 // ------------- END MODULE SCOPE VARIABLES ---------------
 
 // ------------- BEGIN SERVER CONFIGURATION ---------------
-app.configure( function () {
-  app.use( express.bodyParser() );
-  app.use( express.methodOverride() );
+  app.use( bodyParser.urlencoded() );
+  app.use( methodOverride() );
   app.use( express.static( __dirname + '/public' ) );
-  app.use( app.router );
-});
 
-app.configure( 'development', function () {
-  app.use( express.logger() );
-  app.use( express.errorHandler({
-    dumpExceptions : true,
-    showStack      : true
-  }) );
-});
-
-app.configure( 'production', function () {
-  app.use( express.errorHandler() );
-});
+  app.use( morgan( 'common' ) );
+  app.use( errorHandler() );
+//  app.use( errorHandler({
+//  dumpExceptions : true,
+//    showStack      : true
+// });
 
 // all configurations below are for routes
 app.get( '/', function ( request, response ) {
